www/searchutil.php

```diff
@@ -659,6 +661,42 @@ function doSearch($db, $term, $searchType, $sortby, $limit, $browse)
                 }
                 break;
 
+            case 'reviewed':
+                // Only use this query when the user is logged in
+                if ($curuser) {
```

Comment:
I know the rest of the code is like this, but I prefer using "guard"-style checks to avoid extra indentation.
```php
if (!$curuser) {
    break;
}

// rest of code
```